books/ownCloud_Admin_Manual.adoc

```diff
@@ -8,25 +8,32 @@ The ownCloud Team <docs@owncloud.com>
 :toclevels: 2
 :icons: font
 :icon-set: octicon
+:example$: modules/admin_manual/examples
```

Comment:
This is already set within the makefile, so it should be dropped.

Reply:
It was an experiment that didn't work.